katas/content/deutsch_jozsa/index.md

```diff
@@ -141,65 +141,65 @@ The algorithm starts with $N$ qubits in the $\ket{0...0} = \ket{0}^{\otimes N}$
 
-If we apply the $H$ gate to each of the two qubits in the $\ket{00}$ state, we'll get 
+If you apply the $H$ gate to each of the two qubits in the $\ket{00}$ state, you get 
 
 $$(H \otimes H) \ket{00} = \big(H \ket{0} \big) \otimes \big(H \ket{0}\big) = \left(\frac{1}{\sqrt2} \big(\ket{0} + \ket{1} \big)\right) \otimes \left(\frac{1}{\sqrt2} \big(\ket{0} + \ket{1} \big)\right) = \frac{1}{2} \big(\ket{00} + \ket{01} + \ket{10} + \ket{11} \big)$$
 
 This is just an equal superposition of all basis states on two qubits! 
-We can extend the same thinking to applying the $H$ gate to each of the $N$ qubits in the $\ket{0...0}$ state to conclude that this transforms them into a state that is an equal superposition of all basis states on $N$ qubits.
+You can extend the same thinking to applying the $H$ gate to each of the $N$ qubits in the $\ket{0...0}$ state to conclude that this transforms them into a state that is an equal superposition of all basis states on $N$ qubits.
 
-Mathematically the transformation "apply $H$ gate to each of the $N$ qubits" can be denoted as $H^{\otimes N}$. After applying this transformation we'll get the following state:
+Mathematically, the transformation "apply $H$ gate to each of the $N$ qubits" can be denoted as $H^{\otimes N}$. After applying this transformation, you get the following state:
 
 $$H^{\otimes N} \ket{0}^{\otimes N} = \big( H\ket{0} \big)^{\otimes N} = \left( \frac{1}{\sqrt2} \big(\ket{0} + \ket{1} \big) \right)^{\otimes N} = \frac{1}{\sqrt{2^N}} \sum_{x=0}^{2^N-1} \ket{x}$$
 
 
 ### Step 2. Apply the oracle
 
-This step is the only step in which we use the knowledge of the classical function, given to us as the quantum oracle. 
-This step will keep the amplitudes of the basis states for which $f(x) = 0$ unchanged, and multiply the amplitudes of the basis states for which $f(x) = 1$ by $-1$.
+This step is the only step in which you use the knowledge of the classical function, given as the quantum oracle. 
+This step keep the amplitudes of the basis states for which $f(x) = 0$ unchanged, and multiply the amplitudes of the basis states for which $f(x) = 1$ by $-1$.
 
-Mathematically the results of oracle application can be written as follows:
+Mathematically, the results of oracle application can be written as follows:
 
 $$U_f \left(\frac{1}{\sqrt{2^N}} \sum_{x=0}^{2^N-1} \ket{x} \right) = \frac{1}{\sqrt{2^N}} \sum_{x=0}^{2^N-1} U_f\ket{x} = \frac{1}{\sqrt{2^N}} \sum_{x=0}^{2^N-1} (-1)^{f(x)} \ket{x}$$
 
 ### Step 3. Apply Hadamard transform to each qubit again
 
-In this step, let's not worry about the whole expression for the state of the qubits after applying the $H$ gates to them; instead let's calculate only the resulting amplitude of the basis state $\ket{0}^{\otimes N}$.
+In this step, you don't worry about the whole expression for the state of the qubits after applying the $H$ gates to them; instead you calculate only the resulting amplitude of the basis state $\ket{0}^{\otimes N}$.
```

Comment:
This is an instruction for the reader what to do, not a statement of fact, please revert or say "you don't need to worry... it is enough to calculate"